supplements/waterdeep-dungeon-of-the-mad-mage/lycanthrope-werebat.xml

```diff
@@ -0,0 +1,156 @@
+	</element>
+<!-- Echolocation -->
+        <element name="Echolocation" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_ECHOLOCATION">
+		<description>
+			<p>(Bat or Hybrid Form Only). The werebat has blindsight out to a range of 60 feet as long as it's not deafened.</p>
+		</description>
+		<sheet>
+			<description>(Bat or Hybrid Form Only). The werebat has blindsight out to a range of 60 feet as long as it's not deafened.</description>
+		</sheet>
+	</element>
+<!-- Keen Hearing -->
+    <element name="Keen Hearing" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_KEEN_HEARING">
+		<description>
+			<p>The werebat has advantage on Wisdom (Perception) checks that rely on hearing.</p>
+		</description>
+		<sheet>
+			<description>The werebat has advantage on Wisdom (Perception) checks that rely on hearing.</description>
+		</sheet>
+	</element>
+<!-- Nimble Escape -->
+    <element name="Nimble Escape" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_NIMBLE_ESCAPE">
+		<description>
+			<p><b><i>(Humanoid Form Only)</i></b> The werebat can take the Disengage or Hide action as a bonus action on each of its turns.</p>
+		</description>
+		<sheet>
+			<description><b><i>(Humanoid Form Only)</i></b> The werebat can take the Disengage or Hide action as a bonus action on each of its turns.</description>
+		</sheet>
+	</element>
+<!-- Sunlight Sensitivity -->
+    <element name="Sunlight Sensitivity" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_SUNLIGHT_SENSITIVITY">
+        <description>
+            <p>While in sunlight, the werebat has disadvantage on attack rolls, as well as on Wisdom (Perception) checks that rely on sight.. You also have darkvision with a range of 60 ft.</p>
+        </description>
+        <sheet>
+            <description>While in sunlight, the werebat has disadvantage on attack rolls, as well as on Wisdom (Perception) checks that rely on sight.</description>
+        </sheet>
+        <rules>
+            <grant type="Vision" id="ID_VISION_DARKVISION" />
+        </rules>
+    </element>
+<!-- Multiattack -->
+    <element name="Multiattack" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_MULTIATTACK">
+		<description>
+			<p><b><i>(Humanoid or Hybrid Form Only). </i></b>In humanoid form, the werebat makes two scimitar attacks or two shortbow attacks. In hybrid form, it can make one bite attack and one scimitar attack.</p>
+		</description>
+		<sheet action="Action">
+			<description><b><i>(Humanoid or Hybrid Form Only). </i></b>In humanoid form, the werebat makes two light weapon attacks. In hybrid form, it can make one bite attack and one weapon attack.The werebat makes two weapon attacks, one of which can be with its light weapon.</description>
+		</sheet>
```

Comment:
```suggestion
		<sheet action="Action" usage="Humanoid or Hybrid Form">
			<description>In humanoid form, the werebat makes two light weapon attacks. In hybrid form, it can make one bite attack and one weapon attack. The werebat makes two weapon attacks, one of which can be with its light weapon.</description>
		</sheet>
```